### DISCUSSION TITLE
TP3-Ejercicio 1.1

### DISCUSSION THREAD

**Discussion by Clirussa** [INGCOM-UNRN-P1/cursada-2023#125] [Preguntas y respuestas] [answered]
Hola, tenemos una dua con @agosgit  . Estamos arrancando el código de este TP, estuvimos buscando en las consultas pero no nos queda claro:

"1.1 Cargador de arreglos
Desarrollar una función que dado un arreglo de números enteros, y la cantidad de elementos, solicite valores por teclado para llenar las posiciones."

¿Se le indica al usuario que ingrese una cantidad de valores cualquiera? ¿Por que sí? ¿Random? Estuvimos leyendo las preguntas pero no nos queda claro. 

Esperamos ayuda,

**Comment by martinvilu** [comment, marked as answer]
Y, si dice que se ingresa por teclado la consigna, entonces que así sea.

Tal vez sea un procedimiento en lugar de una función
